packages/react/src/Card/CardImage.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes, PropsWithChildren } from 'react'
+import { Image, ImageProps } from '../Image'
+
+export type CardImageProps = ImageProps & PropsWithChildren<HTMLAttributes<HTMLImageElement>>
+
+export const CardImage = forwardRef(
+  ({ children, className, ...restProps }: CardImageProps, ref: ForwardedRef<HTMLImageElement>) => (
+    <Image {...restProps} className={clsx('ams-card__image', className)} ref={ref}>
+      {children}
```

Comment:
An `img` should not have children. I'm not sure why TypeScript didn't catch this

Reply:
Because it’s a valid property – `HTMLImageElement` extends `HTMLElement`. I’ve added #1996 to address this.